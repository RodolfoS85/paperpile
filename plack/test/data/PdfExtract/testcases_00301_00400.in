---
file: ../../../PDFREPOSITORY/000301.pdf
---
file: ../../../PDFREPOSITORY/000302.pdf
---
file: ../../../PDFREPOSITORY/000303.pdf
---
file: ../../../PDFREPOSITORY/000304.pdf
---
file: ../../../PDFREPOSITORY/000305.pdf
---
file: ../../../PDFREPOSITORY/000306.pdf
---
file: ../../../PDFREPOSITORY/000307.pdf
---
file: ../../../PDFREPOSITORY/000308.pdf
---
file: ../../../PDFREPOSITORY/000309.pdf
---
file: ../../../PDFREPOSITORY/000310.pdf
---
file: ../../../PDFREPOSITORY/000311.pdf
---
file: ../../../PDFREPOSITORY/000312.pdf
---
file: ../../../PDFREPOSITORY/000313.pdf
---
file: ../../../PDFREPOSITORY/000314.pdf
---
file: ../../../PDFREPOSITORY/000315.pdf
---
file: ../../../PDFREPOSITORY/000316.pdf
---
file: ../../../PDFREPOSITORY/000317.pdf
---
file: ../../../PDFREPOSITORY/000318.pdf
---
file: ../../../PDFREPOSITORY/000319.pdf
---
file: ../../../PDFREPOSITORY/000320.pdf
---
file: ../../../PDFREPOSITORY/000321.pdf
---
file: ../../../PDFREPOSITORY/000322.pdf
---
file: ../../../PDFREPOSITORY/000323.pdf
---
file: ../../../PDFREPOSITORY/000324.pdf
---
file: ../../../PDFREPOSITORY/000325.pdf
---
file: ../../../PDFREPOSITORY/000326.pdf
---
file: ../../../PDFREPOSITORY/000327.pdf
---
file: ../../../PDFREPOSITORY/000328.pdf
---
file: ../../../PDFREPOSITORY/000329.pdf
---
file: ../../../PDFREPOSITORY/000330.pdf
---
file: ../../../PDFREPOSITORY/000331.pdf
---
file: ../../../PDFREPOSITORY/000332.pdf
---
file: ../../../PDFREPOSITORY/000333.pdf
---
file: ../../../PDFREPOSITORY/000334.pdf
---
file: ../../../PDFREPOSITORY/000335.pdf
---
file: ../../../PDFREPOSITORY/000336.pdf
---
file: ../../../PDFREPOSITORY/000337.pdf
---
file: ../../../PDFREPOSITORY/000338.pdf
---
file: ../../../PDFREPOSITORY/000339.pdf
---
file: ../../../PDFREPOSITORY/000340.pdf
---
file: ../../../PDFREPOSITORY/000341.pdf
---
file: ../../../PDFREPOSITORY/000342.pdf
---
file: ../../../PDFREPOSITORY/000343.pdf
---
file: ../../../PDFREPOSITORY/000344.pdf
---
file: ../../../PDFREPOSITORY/000345.pdf
---
file: ../../../PDFREPOSITORY/000346.pdf
---
file: ../../../PDFREPOSITORY/000347.pdf
---
file: ../../../PDFREPOSITORY/000348.pdf
---
file: ../../../PDFREPOSITORY/000349.pdf
---
file: ../../../PDFREPOSITORY/000350.pdf
---
file: ../../../PDFREPOSITORY/000351.pdf
---
file: ../../../PDFREPOSITORY/000352.pdf
---
file: ../../../PDFREPOSITORY/000353.pdf
---
file: ../../../PDFREPOSITORY/000354.pdf
---
file: ../../../PDFREPOSITORY/000355.pdf
---
file: ../../../PDFREPOSITORY/000356.pdf
---
file: ../../../PDFREPOSITORY/000357.pdf
---
file: ../../../PDFREPOSITORY/000358.pdf
---
file: ../../../PDFREPOSITORY/000359.pdf
---
file: ../../../PDFREPOSITORY/000360.pdf
---
file: ../../../PDFREPOSITORY/000361.pdf
---
file: ../../../PDFREPOSITORY/000362.pdf
---
file: ../../../PDFREPOSITORY/000363.pdf
---
file: ../../../PDFREPOSITORY/000364.pdf
---
file: ../../../PDFREPOSITORY/000365.pdf
---
file: ../../../PDFREPOSITORY/000366.pdf
---
file: ../../../PDFREPOSITORY/000367.pdf
---
file: ../../../PDFREPOSITORY/000368.pdf
---
file: ../../../PDFREPOSITORY/000369.pdf
---
file: ../../../PDFREPOSITORY/000370.pdf
---
file: ../../../PDFREPOSITORY/000371.pdf
---
file: ../../../PDFREPOSITORY/000372.pdf
---
file: ../../../PDFREPOSITORY/000373.pdf
---
file: ../../../PDFREPOSITORY/000374.pdf
---
file: ../../../PDFREPOSITORY/000375.pdf
---
file: ../../../PDFREPOSITORY/000376.pdf
---
file: ../../../PDFREPOSITORY/000377.pdf
---
file: ../../../PDFREPOSITORY/000378.pdf
---
file: ../../../PDFREPOSITORY/000379.pdf
---
file: ../../../PDFREPOSITORY/000380.pdf
---
file: ../../../PDFREPOSITORY/000381.pdf
---
file: ../../../PDFREPOSITORY/000382.pdf
---
file: ../../../PDFREPOSITORY/000383.pdf
---
file: ../../../PDFREPOSITORY/000384.pdf
---
file: ../../../PDFREPOSITORY/000385.pdf
---
file: ../../../PDFREPOSITORY/000386.pdf
---
file: ../../../PDFREPOSITORY/000387.pdf
---
file: ../../../PDFREPOSITORY/000388.pdf
---
file: ../../../PDFREPOSITORY/000389.pdf
---
file: ../../../PDFREPOSITORY/000390.pdf
---
file: ../../../PDFREPOSITORY/000391.pdf
---
file: ../../../PDFREPOSITORY/000392.pdf
---
file: ../../../PDFREPOSITORY/000393.pdf
---
file: ../../../PDFREPOSITORY/000394.pdf
---
file: ../../../PDFREPOSITORY/000395.pdf
---
file: ../../../PDFREPOSITORY/000396.pdf
---
file: ../../../PDFREPOSITORY/000397.pdf
---
file: ../../../PDFREPOSITORY/000398.pdf
---
file: ../../../PDFREPOSITORY/000399.pdf
---
file: ../../../PDFREPOSITORY/000400.pdf
