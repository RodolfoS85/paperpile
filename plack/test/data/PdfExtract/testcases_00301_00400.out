---
authors: 'DEL GATTO, Fabienne and Plet, Ariane and Gesnel, MARIE-CLAUDE and Fort, Cecile and Breathnach, Richard'
title: Multiple Interdependent Sequence Elements Control Splicing of a Fibroblast Growth Factor Receptor 2 Alternative Exon
---
authors: 'Delcher, Arthur L. and Kasif, Simon and Fleischmann, Robert D. and Peterson, Jeremy and White, Owen and Salzberg, Steven L.'
title: Alignment of whole genomes
---
authors: 'Delihas, Nicholas'
title: Annotation and evolutionary relationships of a small regulatory RNA gene micF and its target ompF in Yersinia species
---
authors: 'Delihas, Nicholas and Forst, Steven'
doi: 10.1006/jmbi.2001.5029
title: 'MicF: An Antisense RNA Gene Involved in Response of Escherichia coli to Global Stress Factors'
---
authors: 'Delihas, Nicholas'
doi: 10.1186/1471-2164-10-101
title: Intergenic regions of Borrelia plasmids contain phylogenetically conserved RNA secondary structure motifs
---
authors: 'Dellaert, Frank and Fox, Dieter and Burgard, Wolfram and Thrun, Sebastian'
title: Monte Carlo Localization for Mobile Robots
---
authors: 'Delneri, Daniela and Brancia, Francesco L and Oliver, Stephen G'
title: Towards a truly integrative biology through the functional genomics of yeast
---
authors: 'Dember, Laura M. and Kim, Nancy D. and Liu, Karen-Qianye and Anderson, Paul'
title: Individual RNA Recognition Motifs of TIA-1 and TIAR Have Different RNA Binding Specificities
---
authors: 'Dermitzakis, Emmanouil T. and Clark, Andrew G.'
title: 'Evolution of Transcription Factor Binding Sites in Mammalian Gene Regulatory Regions: Conservation and Turnover'
---
authors: 'Dermitzakis, Emmanouil T. and Reymond, Alexandre and Scamuffa, Nathalie and Ucla, Catherine and Kirkness, Ewen and Rossier, Colette and Antonarakis, Stylianos E.'
title: Evolutionary Discrimination of Mammalian Conserved Non-Genic Sequences (CNGs)
---
authors: 'Desjarlais, John R and Clarke, Neil D'
title: Computer search algorithms in protein modification and design
---
authors: 'Deutsch, David'
title: 'Quantum theory, the Church-Turing principle and the universal quantum computer'
---
authors: 'Dhaeseleer, Patrik'
title: How does DNA sequence motif discovery work?
---
authors: 'Dhaeseleer, Patrik'
title: What are DNA sequence motifs?
---
authors: 'Dhar, Pawan K. and Zhu, Hao and Mishra, Santosh K.'
doi: 10.1109/TNB.2004.833699
title: 'Computational Approach to Systems Biology: From Fraction to Integration and Beyond'
---
authors: 'Dias, Anusha P. and Braun, Edward L. and McMullen, Michael D. and Grotewold, Erich'
doi: 10.1104/pp.012047
title: Recently Duplicated Maize R2R3 Myb Genes Provide Evidence for Distinct Mechanisms of Evolutionary Divergence after Duplication
---
authors: 'Dickerson, Richard E.'
title: 'DNA bending: the prevalence of kinkiness and the virtues of normality'
---
authors: 'Diella, Francesca and Cameron, Scott and Gemünd, Christine and Linding, Rune and Via, Allegra and Kuster, Bernhard and Sicheritz-Pontén, Thomas and Blom, Nikolaj and Gibson, Toby J'
doi: 10.1186/1471-2105-5-79
title: 'Phospho.ELM: A database of experimentally verified phosphorylation sites in eukaryotic proteins'
---
authors: 'Dieterich, Christoph and Cusack, Brian and Wang, Haiyan and Rateitschak, Katja and Krause, Antje and Vingron, Martin'
title: Annotating regulatory DNA based on man-mouse genomic comparison
---
authors: 'Dietrich, Rosemary C. and Fuller, John D. and Padgett, Richard A.'
doi: 10.1261/rna.7206305
title: A mutational analysis of U12-dependent splice site dinucleotides
---
authors: 'Dietrich, Rosemary C. and Shukla, Girish C. and Fuller, John D. and Padgett, Richard A.'
title: Alternative splicing of U12-dependent introns in vivo responds to purine-rich enhancers
---
authors: 'Dimitrov, Roumen A. and Zuker, Michael'
doi: 10.1529/biophysj.103.020743
title: Prediction of Hybridization and Melting for Double-Stranded Nucleic Acids
---
authors: 'Ding, YE and Chan, Chi YU and Lawrence, Charles E.'
doi: 10.1261/rna.2500605
title: RNA secondary structure prediction by centroids in a Boltzmann weighted ensemble
---
authors: 'Ding, Ye and Lawrence, Charles E.'
title: Statistical prediction of single-stranded regions in RNA secondary structure and application to predicting effective antisense target sites and beyond
---
authors: 'Ding, Chris and Peng, Hanchuan'
title: Minimum Redundancy Feature Selection from Microarray Gene Expression Data
---
authors: 'Dinger, Marcel E. and Pang, Ken C. and Mercer, Tim R. and Mattick, John S.'
doi: 10.1371/journal.pcbi.1000176
title: 'Differentiating Protein-Coding and Noncoding RNA: Challenges and Ambiguities'
---
authors: 'Dinner, Aaron R. and Sali, Andrej and Karplus, Martin'
title: 'The folding mechanism of larger model proteins: Role of native structure'
---
authors: 'Dinner, Aaron R. and Sali, Andrej and Smith, Lorna J. and Dobson, Christopher M. and Karplus, Martin'
title: Understanding protein folding via free-energy surfaces from theory and experiment
---
authors: 'Dirks, Robert M. and Bois, Justin S. and Schaeffer, Joseph M. and Winfree, Erik and Pierce, Niles A.'
doi: 10.1137/060651100
title: Thermodynamic Analysis of Interacting Nucleic Acid Strands
---
authors: 'Dirks, Robert M. and Pierce, Niles A.'
title: A Partition Function Algorithm for Nucleic Acid Secondary Structure Including Pseudoknots
---
authors: 'Djordjevic, Marko and Sengupta, Anirvan M. and Shraiman, Boris I.'
doi: 10.1101/gr.1271603
title: A Biophysical Approach to Transcription Factor Binding Site Discovery
---
authors: 'Dlakic, Mensur and Harrington, Rodney E.'
title: 'DIAMOD: display and modeling of DNA bending'
---
authors: 'Doebley, John and Lukens, Lewis'
title: Transcriptional Regulators and the Evolution of Plant Form
---
authors: 'Doench, John G. and Sharp, Phillip A.'
doi: 10.1101/gad.1184404
title: Specificity of microRNA target selection in translational repression
---
authors: 'Dorogovtsev, S. N. and Mendes, J. F. F.'
title: Evolution of networks with aging of sites
---
authors: 'Doruker, Pemra and Mattice, Wayne L.'
title: Reverse Mapping of Coarse-Grained Polyethylene Chains from the Second Nearest Neighbor Diamond Lattice to an Atomistic Model in Continuous Space
---
authors: 'Dotu, Ivan'
title: RNA STRUCTURAL SEGMENTATION
---
authors: 'Douchin, Veronique and Bohn, Chantal and Bouloc, Philippe'
doi: 10.1074/jbc.M600819200
title: Down-regulation of Porins by a Small RNA Bypasses the Essentiality of the Regulated Intramembrane Proteolysis Protease RseP in Escherichia coli
---
authors: 'Doudna, Jennifer A. and Cech, Thomas R.'
title: The chemical repertoire of natural ribozymes
---
authors: 'Dowell, Robin D and Eddy, Sean R'
doi: 10.1186/1471-2105-7-400
title: Efficient pairwise RNA structure prediction and alignment using sequence alignment constraints
---
authors: 'Dowell, Robin D and Eddy, Sean R'
title: Evaluation of several lightweight stochastic context-free grammars for RNA secondary structure prediction
---
authors: 'Downey, Rodney G. and Fellows, Michael R.'
title: Fixed-Parameter Intractability
---
authors: 'Dralyuk, I. and Brudno, M. and Gelfand, M. S. and Zorn, M. and Dubchak, I.'
title: 'ASDB: database of alternatively spliced genes'
---
authors: 'Dress, Andreas'
title: Proper Gromov Transforms of Metrics are Metrics
---
authors: 'Dreumont, Natacha and Maresca, Antonella and Boisclair-Lachance, Jean-François and Bergeron, Anne and Tanguay, Robert M'
doi: 10.1186/1471-2199-6-1
title: A minor alternative transcript of the fumarylacetoacetate hydrolase gene produces a protein despite being likely subjected to nonsense-mediated mRNA decay
---
authors: 'Duchange, Nathalie and Saleh, Marla-Carla and de Arriba Zerpa, Gonzalo and Pidoux, Josette and Guillou, Florian and Zakin, Mario M. and Baron, Bruno'
title: 'Alternative Splicing in the Brain of Mice and Rats Generates Transferrin Transcripts Lacking, as in Humans, the Signal Peptide Sequence'
---
authors: 'Duhring, Ulf and Axmann, Ilka M. and Hess, Wolfgang R. and Wilde, Annegret'
doi: 10.1073/pnas.0600927103
title: An internal antisense RNA regulates expression of the photosynthesis gene isiA
---
authors: 'Echard, Arnaud and Opdam, Frank J. M. and de Leeuw, Hubert J. P. C. and Jollivet, Florence and Savelkoul, Paul and Hendriks, Wiljan and Voorberg, Jan and Goud, Bruno and Fransen, Jack A. M.'
title: Alternative Splicing of the Human Rab6A Gene Generates Two Close but Functionally Different Isoforms
---
authors: 'Eddy, Sean R'
title: A memory-efficient dynamic programming algorithm for optimal alignment of a sequence to an RNA secondary structure
---
authors: 'Eddy, Sean R.'
doi: 10.1371/journal.pbio.0030010
title: A Model of the Statistical Power of Comparative Genome Sequence Analysis
---
authors: 'Eddy, Sean R'
title: What is dynamic programming?
---
authors: 'Eddy, Sean R'
title: Where did the BLOSUM62 alignment score matrix come from?
---
authors: 'Edgar, Robert C'
doi: 10.1186/1471-2105-5-113
title: 'MUSCLE: a multiple sequence alignment method with reduced time and space complexity'
---
authors: 'Eidhammer, Ingvar and Jonassen, Inge and Grindhaug, Svenn Helge and inge, ingvar'
title: A Constraint Based Structure Description Language for Biosequences
---
authors: 'Isen, M Ichael B. E and Spellman, Paul T. and Brown, Patrick O. and Botstein, David'
title: Cluster analysis and display of genome-wide expression patterns
---
authors: 'El-Mabrouk, Nadia and Sankoff, David'
title: On the Reconstruction of Ancient Doubled Circular Genomes Using Minimum Reversals
---
authors: 'El Hassan, M. A. and Calladine, C. R.'
title: Two Distinct Modes of Protein-induced Bending in DNA
---
authors: 'Elango, Navin and Thomas, James W. and Program, Nisc Comparative Sequencing and Yi, Soojin V.'
doi: 10.1073/pnas.0510716103
title: Variable molecular clocks in hominoids
---
authors: 'Ellrott, Kyle and Yang, Chuhu and Sladek, Frances M. and Jiang, Tao'
title: Identifying transcription factor binding sites through Markov chain optimization
---
authors: 'Eres, Revital and Landau, Gad M. and Parida, Laxmi'
title: Permutation Pattern Discovery in Biosequences
---
authors: 'Eng, Laura and Coutinho, Gabriela and Nahas, Shareef and Yeo, Gene and Tanouye, Robert and Babaei, Mahnoush and Dork, Thilo and Burge, Christopher and Gatti, Richard A.'
doi: 10.1002/humu.10295
title: 'Nonclassical Splicing Mutations in the Coding and Noncoding Regions of the ATM Gene: Maximum Entropy Estimates of Splice Junction Strengths'
---
authors: 'England, Jeremy L. and Shakhnovich, Eugene I.'
doi: 10.1103/PhysRevLett.90.218101
title: Structural Determinant of Protein Designability
---
authors: 'Enright, Anton J and John, Bino and Gaul, Ulrike and Tuschl, Thomas and Sander, Chris and Marks, Debora S'
title: MicroRNA targets in Drosophila
---
arxivid: cs.DS/0009006
authors: 'Eppstein, David'
title: 'Improved Algorithms for 3-Coloring, 3-Edge-Coloring, and Constraint Satisfaction'
---
authors: 'Epstein, Susan L. and Freuder, Eugene C. and Wallace, Richard and Morozov, Anton and Samuels, Bruce'
title: The Adaptive Constraint Engine
---
authors: 'Ermolaeva, Maria D. and Khalak, Hanif G. and White, Owen and Smith, Hamilton O. and Salzberg, Steven L.'
doi: 10.1006/jmbi.2000.3836
title: Prediction of Transcription Terminators in Bacterial Genomes
---
authors: 'Eskin, Eleazar and Pevzner, Pavel A.'
title: Finding Composite Regulatory Patterns in DNA Sequences
---
authors: 'Eswarakumar, Vereragavan P. and Monsonego-Ornan, Efrat and Pines, Mark and Antonopoulou, Ileana and Morriss-Kay, Gillian M. and Lonai, Peter'
title: The IIIc alternative of Fgfr2 is a positive regulator of bone formation
---
authors: 'Evans, Patricia A.'
title: Finding Common RNA Pseudoknot Structures in Polynomial Time
---
authors: 'Evans, Patricia A.'
title: Finding Common Subsequences with Arcs and Pseudoknots
---
authors: 'Evers, Dirk and Giegerich, Robert'
title: 'RNA Movies: visualizing RNA secondary structure spaces'
---
authors: 'Fabre-Suver, Christine and Hauschka, Stephen D.'
title: A Novel Site in the Muscle Creatine Kinase Enhancer Is Required for Expression in Skeletal but Not Cardiac Muscle
---
authors: 'Fagegaltier, Delphine and Hubert, Nadia and Carbon, Philippe and Krol, Alain'
title: The selenocysteine insertion sequence binding protein SBP is different from the Y-box protein dbpB
---
authors: 'Fagegaltier, Delphine and Hubert, Nadia and Yamada, Kenichiro and Mizutani, Takaharu and Carbon, Philippe and Krol, Alain'
title: 'Characterization of mSelB, a novel mammalian elongation factor for selenoprotein translation'
---
authors: 'Fagegaltier, Delphine and Lescure, Alain and Walczak, Robert and Carbon, Philippe and Krol, Alain'
title: Structural analysis of new local features in SECIS RNA hairpins
---
authors: 'Fairbrother, William G. and Chasin, Lawrence A.'
title: Human Genomic Sequences That Inhibit Splicing
---
authors: 'Faith, Jeremiah J. and Hayete, Boris and Thaden, Joshua T. and Mogno, Ilaria and Wierzbowski, Jamey and Cottarel, Guillaume and Kasif, Simon and Collins, James J. and Gardner, Timothy S.'
doi: 10.1371/journal.pbio.0050008
title: Large-Scale Mapping and Validation of Escherichia coli Transcriptional Regulation from a Compendium of Expression Profiles
---
authors: 'Famulok, M. and Mayer, G.'
title: Aptamers as Tools in Molecular Biology and Immunology
---
authors: 'Farabaugh, P. J.'
title: PROGRAMMED TRANSLATIONAL FRAMESHIFTING
---
authors: 'Farabaugh, Philip J.'
title: Programmed Translational Frameshifting
---
authors: 'Farrance, Iain K. G. and Ordahl, Charles P.'
title: The Role of Transcription Enhancer Factor-1 (TEF-1) Related Proteins in the Formation of M-CAT Binding Complexes in Muscle and Non-muscle Tissues
---
authors: 'Fattash, Isam and Voß, Björn and Reski, Ralf and Hess, Wolfgang R and Frank, Wolfgang'
doi: 10.1186/1471-2229-7-13
title: Evidence for the rapid expansion of microRNA-mediated regulation in early land plant evolution
---
authors: 'Fededa, Juan P. and Petrillo, Ezequiel and Gelfand, Mikhail S. and Neverov, Alexei D. and Kadener, Sebastián and Nogués, Guadalupe and Pelisch, Federico and Baralle, Francisco E.'
doi: 10.1016/j.molcel.2005.06.035
title: A Polar Mechanism Coordinates Different Regions of Alternative Splicing within a Single Gene
---
authors: 'Fedor, Martha J.'
doi: 10.1006/jmbi.2000.3560
title: Structure and Function of the Hairpin Ribozyme
---
authors: 'Fedorov, Alexei and Cao, Xiaohong and Saxonov, Serge and de Souza, Sandro J. and Roy, Scott W. and Gilbert, Walter'
doi: 10.1073/pnas.231491498
title: Intron distribution difference for 276 ancient and 131 modern genes suggests the existence of ancient introns
---
authors: 'Fedorov, Alexey and Fedorova, Larisa and Starshenko, Valery and Filatov, Vadim and Grigorev, Eugeni'
title: Influence of Exon Duplication on Intron and Exon Phase Distribution
---
authors: 'Fedorov, Alexei and Saxonov, Serge and Fedorova, Larisa and Daizadeh, Iraj'
title: Comparison of intron-containing and intron-lacking human genes elucidates putative exonic splicing enhancers
---
authors: 'Fedorova, Larisa and Fedorov, Alexei'
title: Introns in gene evolution
---
authors: 'Felden, Brice'
doi: 10.1016/j.mib.2007.05.001
title: 'RNA structure: experimental analysis'
---
authors: 'Fenyö, David'
title: 'Identifying the proteome: software tools'
---
authors: 'Ferbeyre, Gerardo and Bourdeau, Veronique and Pageau, Marie and Miramontes, Pedro and Cedergren, Robert'
title: Distribution of Hammerhead and Hammerhead-like RNA Motifs Through the GenBank
---
authors: 'Fersht, Alan R. and Daggett, Valerie'
title: Protein Folding and Unfolding at Atomic Resolution
---
authors: 'Fickett, James W. and Hatzigeorgiou, Artemis G.'
title: Eukaryotic Promoter Recognition
---
authors: 'Fickett, James W.'
title: Quantitative Discrimination of MEF2 Sites
---
authors: 'Fiegen, Dennis and Haeusler, Lars-Christian and Blumenstein, Lars and Herbrand, Ulrike and Dvorsky, Radovan and Vetter, Ingrid R. and Ahmadian, Mohammad R.'
doi: 10.1074/jbc.M310281200
title: 'Alternative Splicing of Rac1 Generates Rac1b, a Self-activating GTPase'
---
authors: 'Filbin, Megan E and Kieft, Jeffrey S'
doi: 10.1016/j.sbi.2009.03.005
title: Toward a structural understanding of IRES RNA function
---
authors: 'Fjose, Anders and Drivenes, Øyvind'
doi: 10.1002/bdrc.20069
title: 'RNAi and MicroRNAs: From Animal Models to Disease Therapy'
---
authors: 'Flamm, Christoph and Fontana, Walter and Hofacker, Ivo L. and Schuster, Peter'
title: RNA folding at elementary step resolution
---
authors: 'Flamm, Christoph and Hofacker, Ivo L.'
title: 'Beyond Energy Minimization: Approaches to the kinetic Folding of RNA'
---
authors: 'Florea, Liliana and Hartzell, George and Zhang, Zheng and Rubin, Gerald M. and Miller, Webb'
title: A Computer Program for Aligning a cDNA Sequence with a Genomic DNA Sequence
